compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/Transforms.cpp

```diff
@@ -620,16 +620,16 @@ distributeMultiMmaOp(RewriterBase &rewriter, IREE::GPU::MultiMmaOp mmaOp,
       accStrides);
 
   // Step 3. Create the new multi_mma op.
-  auto newKind = mmaOp.getKind();
-  if (auto dataTiledMma = dyn_cast<DataTiledMMAAttr>(newKind)) {
-    newKind = DataTiledMMAAttr::get(
-        context, dataTiledMma.getIntrinsic(), dataTiledMma.getUnrollM(),
-        /*subgroups_m=*/1, dataTiledMma.getUnrollN(),
-        /*subgroups_n=*/1, dataTiledMma.getUnrollK());
-  }
   auto newMmaOp = rewriter.create<IREE::GPU::MultiMmaOp>(
       loc, lhsSlice, rhsSlice, accSlice, mmaOp.getIndexingMaps(),
-      mmaOp.getIteratorTypes(), newKind);
+      mmaOp.getIteratorTypes(), mmaOp.getKind());
+
+  // Preserve the lowering_config attribute for GPULowerToUKernelsPass.
+  constexpr char loweringConfigAttrName[] = "lowering_config";
+  if (mmaOp->hasAttr(loweringConfigAttrName)) {
+    newMmaOp->setAttr(loweringConfigAttrName,
+                      mmaOp->getAttr(loweringConfigAttrName));
+  }
```

Comment:
nit: use `kConfigAttrName`.

https://github.com/iree-org/iree/blob/e55342565017e86944ce57d593672e5bfb690a05/compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.h#L47

Question: do we preserve all the discardable attributes (i.e., additional attributes that are not defined by the op itself)? If so, you can do something like

```
newMmaOp->setDiscardableAttrs(mmaOp->getDiscardableAttrDictionary());
```

Reply:
Thanks, that worked! Earlier I had tried something with `setAttrs` / `getAttrs` and that caused other tests to fail. I didn't know about `setDiscardableAttrs`.